Formula/cadence-workflow.rb

```diff
@@ -0,0 +1,33 @@
+class CadenceWorkflow < Formula
+  desc "Cadence Workflow is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way"
+  homepage "https://cadenceworkflow.io/"
+  url "https://github.com/uber/cadence.git",
+      tag:      "v0.18.1",
+      revision: "b6dfe37ebfec7742b1470be7129f07518e8bec98"
+  license "MIT"
+
+  bottle do
+  end
+
+  depends_on "go" => :build
+
+  conflicts_with "cadence", because: "both install an `cadence` executable"
+
+  def install
+    ENV["PATH"] = "#{ENV["PATH"]}:#{ENV["GOPATH"]}/bin"
+    system "make", "cadence", "cadence-server", "cadence-canary", "cadence-sql-tool", "cadence-cassandra-tool"
```

Comment:
Why can't we use the default `make` target?

Reply:
Because the default target doesn't do anything:
```
qlong@~/cadence:
(master)$make
help                 Prints a help message showing any specially-commented targets
clean                Clean binaries and build folder
copyright            update copyright headers
deps                 Check for dependency updates, for things that are directly imported
deps-all             Check for all dependency updates
fmt                  run goimports
lint                 (re)run the linter
release              Re-generate generated code and run tests
test                 Build and run all tests
qlong@~/cadence:
```

Reply:
Actually we could do `make bins`. Does it look better for you?